src/components/widgets/Bolk/ListeItemBolk.jsx

```diff
@@ -7,7 +7,7 @@ import './ListeItemBolk.css';
 const ListeItemBolk = ({ label, value }) => (
     <span className="ListeItemBolk">
         <ListeItem label={label} value={value} />
-        <EnigBoks />
+        <EnigBoks id={label} />
```

Comment:
Vil denne være unik nok? Vil det ikke bli problemer med labler som feks. "Fom."?

Reply:
Jeg kan ikke se at det er noen labels saksbehandler kan være "uenig" i som forekommer to ganger. Vet ikke om de trenger være unike uansett? Jeg tenker at det viktigste er at hver "uenighet" det rapporteres om har en beskrivende label, som i dette tilfellet blir id-propen. Kanskje rename den til `label`?

Reply:
Ja, det var blant annet future proofing jeg tenkte på.. Hadde potensial til å bli en obskur bug.